src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreCommand.cs

```diff
@@ -77,6 +77,7 @@ private async Task SubscribeAsync(Microsoft.VisualStudio.Shell.IAsyncServiceProv
             commandService.AddCommand(menuItem);
 
             _vsMonitorSelection = await serviceProvider.GetServiceAsync(typeof(IVsMonitorSelection)) as IVsMonitorSelection;
+            Assumes.Present(_vsMonitorSelection);
```

Comment:
Do we need more `Assumes.Present()`  for all the services used in this class?

Reply:
I am addressing the "request a service" -> "use a service" pattern flagged by the analyzer. 

I wouldn't want to claim that everything is properly null checked, because that's not what I'm trying to address :)